apps/web/src/components/tx-flow/common/TxLayout/styles.module.css

```diff
@@ -11,8 +11,7 @@
   display: flex;
   justify-content: space-between;
   align-items: center;
-  /* Remove height of progress bar */
-  padding: calc(var(--space-3) - 6px) var(--space-3) var(--space-3);
```

Comment:
This made the header's paddings look disproportionate. The progress bar is outside of the block and shouldn't be compensated for.